benchmarks/benchmark_common.h

```diff
@@ -0,0 +1,46 @@
+#ifndef BENCHMARK_COMMON_I
+#define BENCHMARK_COMMON_I
+
+#include <random>
+
+namespace benchmark_common {
+
+enum class Cplx { EXT, STD };
+
+template <Cplx cplx, typename R>
+using complex_t =
+    typename std::conditional_t<cplx == Cplx::EXT, sycl::ext::cplx::complex<R>,
+                                std::complex<R>>;
+
+template <typename R, std::uint32_t SEED = 777>
+inline void fill_random(R *data, size_t n) {
+  std::uniform_real_distribution<R> dist(-1.0, 1.0);
```

Comment:
-1, 1. seem pretty small range for math function no?

Reply:
I changed to -100, 100 for now. I think ideally it would take range as parameter, because for periodic functions something like [-pi, pi] would be enough and others would benefit from very larger range.

Reply:
Agree, it will be better to change the range according to the function.  
Trig will need -pi , pi, or -pi/2, pi/2 stuff like that.

Or for pow, we may need a big range.  But we can improve later